crates/wast-util/src/lib.rs

```diff
@@ -526,6 +525,16 @@ impl WastTest {
             if unsupported.iter().any(|part| self.path.ends_with(part)) {
                 return true;
             }
+
+            // SIMD on Winch requires AVX instructions.
+            #[cfg(target_arch = "x86_64")]
+            if !(std::is_x86_feature_detected!("avx") && std::is_x86_feature_detected!("avx2")) {
+                let unsupported = ["spec_testsuite/simd_align.wasm"];
```

Comment:
```suggestion
                let unsupported = ["spec_testsuite/simd_align.wast"];
```

This is why the PR is still failing in CI. I tested the change in my arm64 machine FWIW.